miden/src/cli/run.rs

```diff
@@ -40,6 +40,10 @@ pub struct RunCmd {
     /// Enable tracing to monitor execution of the VM
     #[clap(short = 't', long = "tracing")]
     tracing: bool,
+
+    /// Enable debug instructions
+    #[clap(short = 'd', long = "debug")]
+    debug: bool,
```

Comment:
The naming isn't quite parallel here. So, I'm wondering if we should rename `tracing` to `trace` (otherwise it is a bit confusing why it is `tracing` but not `debugging` and I think `trace`/`debug` is better than `tracing`/`debugging`).

Reply:
@bobbinth Would you want to change both the variable and the flag?

```
#[clap(short = 't', long = "trace")]
trace: bool,
```
Also that would be a breaking change and need a separate entry in the change log, right?

Reply:
Yep, let's change the variable too and yes, would probably mention in the changelog as well.